app/assets/javascripts/discourse/tests/acceptance/sidebar-narrow-desktop-test.js

```diff
@@ -55,4 +55,30 @@ acceptance("Sidebar - Narrow Desktop", function (needs) {
 
     bodyElement.style.width = null;
   });
+
+  test("transition from narrow screen to wide screen", async function (assert) {
+    await visit("/");
+    await settled();
+
+    const bodyElement = document.querySelector("body");
+    bodyElement.style.width = "990px";
+
+    await waitUntil(
+      () => document.querySelector(".btn-sidebar-toggle.narrow-desktop"),
+      {
+        timeout: 5000,
+      }
+    );
+    await click(".btn-sidebar-toggle");
+
+    bodyElement.style.width = "1200px";
+    await waitUntil(() => document.querySelector("#d-sidebar"), {
+      timeout: 5000,
+    });
+    await click(".header-dropdown-toggle.current-user");
+    $(".header-dropdown-toggle.current-user").click();
+    assert.ok(exists(".quick-access-panel"));
+
+    bodyElement.style.width = null;
```

Comment:
What happens when width is set to `null`? I'm thinking the body's width should always be set back to the original width before this test to ensure we're not changing any state.

Reply:
It clears inline style, after `bodyElement.style.width = "490px"` we have

<img width="554" alt="Screenshot 2022-12-05 at 12 34 02 pm" src="https://user-images.githubusercontent.com/72780/205530288-e14f2e1d-2af6-45ea-9f17-2d9cdb1a51c7.png">

And after `bodyElement.style.width = null` we have
<img width="553" alt="Screenshot 2022-12-05 at 12 34 24 pm" src="https://user-images.githubusercontent.com/72780/205530368-1d6c9986-dc18-4b70-bf96-36d268374f96.png">

Reply:
:+1: